examples/NavigationPlayground/flow-typed/npm/expo_vx.x.x.js

```diff
@@ -254,6 +314,14 @@ declare module 'expo/src/sensor/__tests__/Accelerometer-test' {
   declare module.exports: any;
 }
 
+declare module 'expo/src/sensor/__tests__/DeviceMotion-test' {
```

Comment:
why would expo's tests need to be flow-typed?

Reply:
`flow-typed` creates these autogenerated stubs for any library it doesn't have a libdef for. I'm not entirely sure why, but the comment in the file seems to indicate it's to help people start the "gradual typing" process